uiowa/uiowa
# [Classrooms] a11y filter

Resolves https://github.com/uiowa/uiowa/issues/6560

This pr moves `Better Exposed filter` and `Calendar` styles to their own libraries and I have attached those libraries to each site using those modules instead of loading all of that CSS for any page with a view. 

# How to test

```
ddev blt frontend && ddev blt ds --site=classrooms.uiowa.edu && ddev blt ds --site=admissions.uiowa.edu && ddev blt ds --site=housing.uiowa.edu && ddev blt ds --site=iisc.uiowa.edu && ddev blt ds --site=ighn.international.uiowa.edu && ddev blt ds --site=iowasummerwritingfestival.uiowa.edu && ddev blt ds --site=sandbox.uiowa.edu && ddev blt ds --site=grad.admissions.uiowa.edu && ddev blt ds --site=education.uiowa.edu && ddev blt ds --site=grad.uiowa.edu &&  ddev drush @classrooms.local uli find-university-classroom && ddev drush @admissions.local uli areas-of-study && ddev drush @housing.local uli residence-halls && ddev drush @iisc.local uli projects && ddev drush @internationalighn.local uli home && ddev drush @iowasummerwritingfestival.local uli events && ddev drush @sandbox.local uli people && ddev drush @admissionsgrad.local uli areas-of-study && ddev drush @education.local uli areas-study/program-finder && ddev drush @grad.local uli academics/thesis-defenses 
```



Test new filter and layout on:
- https://classrooms.uiowa.ddev.site/find-university-classroom
  - Does it work?
  - Are items listed alphabetical?
  - Can you select multiple items for filtering?
  - Check mobile and desktop functionality
  - Check result count position - is it under the pager?
  - Test checkboxes
  - Test submit and reset filters at all breakpoints
  - Test "filters" toggle open/close spacing on mobile and desktop
- Test for regressions on https://classrooms.uiowa.ddev.site/spaces/programmed-classrooms. 

## Other sites affected by these style changes

Test for:

- Visual regresssions
- Test checkboxes
- Test submit and reset filters at all breakpoints
- Test "filters" toggle open/close spacing on mobile and desktop


Urls to check:
- https://iowasummerwritingfestival.uiowa.ddev.site/events
- https://oniowa.uiowa.ddev.site/events
- https://admissions.uiowa.ddev.site/events/calendar
- https://admissions.uiowa.ddev.site/events
- https://admissions.uiowa.ddev.site/areas-of-study
- https://admissions.uiowa.ddev.site/counselors
- https://housing.uiowa.ddev.site/residence-halls
- https://iisc.uiowa.ddev.site/projects
- https://internationalighn.uiowa.ddev.site/
- https://sandbox.uiowa.ddev.site/people
- https://grad.uiowa.ddev.site/academics/thesis-defenses
- https://education.uiowa.ddev.site/areas-study/program-finder
